app/template/api/index.tsx

```diff
@@ -0,0 +1,13 @@
+const getFromApiClientSide = async (): Promise<Response> => {
```

Comment:
I originally wrote some comments here they're better placed here https://github.com/ed-pilots-network/frontend/pull/119/files#diff-228a314c046a3a9b4c3e7ff6bdf0dc14c18826b9e0eea866b4def74d54b2e1dfR10 and both sort of tie in with each other